KABOT-Project/Scratch-Project
# Bl/react basics

##Overview

**Issue Type**
-[] Bug
-[X] Feature
-[]Tech debt
'
**Description**
Existing recipes are now rendered to Recipes page upon initial render (using dummy data)

**Steps to Reproduce Bug/Validate Feature/Confirm Tech Debt Fix**
Go to http://localhost:8080/recipes
See 4 recipe names rendered on the page (Pasta, Burger, Ice Cream, Seafood)

**Previous behavior**
NA

**Expected behavior**
Dummy recipe names rendered on page

**Screenshots & Videos**
<img width="266" alt="Screenshot 2023-07-08 at 2 37 20 PM" src="https://github.com/KABOT-Project/Scratch-Project/assets/121215062/763c5d4a-acf5-4cd2-a468-2b8d32a8c89b">


**Additional context**
NA
